### ISSUE TITLE
I'm looking forward to exposing the best models you've trained

### ISSUE THREAD

**Issue by BeiZhiRen** [CUHK-AIM-Group/U-KAN#4]
This is a very meaningful split network structure, thank you very much for the work you and your team have done, but I have a doubt, I tried to use an NVIDIA GeForce RTX 3090 training GlaS dataset, the dataset distribution and parameter settings are consistent with your task, but my IOU of the best result of training 400 rounds is only 84.55, which is far lower than the IOU result given by your paper (87.64), I wonder if you and your team can answer my doubts, Can you also give you the best model you trained?

**Comment by BeiZhiRen** [comment]
python train.py --arch UKAN --dataset glas --input_w 256 --input_h 256 --name glas_UKAN --data_dir ./inputs

**Comment by XGGNet** [comment]
Thank you for your interest! It is likely that the learning rate --lr was set incorrectly. 

The learning rate reported in the paper is 1e-4, but the default learning rate in the initial version of the code mistakenly set as 1e-2 (this has now been corrected to 1e-4). It should affect a lot on performance. I will endeavor to confirm this promptly and engage in further communication with you. 

As for pre-trained models, we will try best to handle it in these days.

**Comment by XGGNet** [comment]
> python train.py --arch UKAN --dataset glas --input_w 256 --input_h 256 --name glas_UKAN --data_dir ./inputs

Another issue is the incorrect setting of resolution. It is 512 rather than 256. The following example script can be found at Seg_UKAN/scripts.sh:

dataset=glas
input_size=512
python train.py --arch UKAN --dataset ${dataset} --input_w ${input_size} --input_h ${input_size} --name ${dataset}_UKAN  --data_dir [YOUR_DATA_DIR]
python val.py --name ${dataset}_UKAN

**Comment by BeiZhiRen** [comment]
> > python train.py --arch UKAN --dataset glas --input_w 256 --input_h 256 --name glas_UKAN --data_dir ./inputs
> 
> 另一个问题是分辨率设置不正确。它是 512 而不是 256。可以在 Seg_UKAN/scripts.sh 找到以下示例脚本：
> 
> dataset=glas input_size=512 python train.py --arch UKAN --dataset ${dataset} --input_w ${input_size} --input_h ${input_size} --name ${dataset}_UKAN --data_dir [YOUR_DATA_DIR] python val.py --name ${dataset}_UKAN

Oh, I'm sorry, this is my problem, I'll try again after revising, thank you very much for your reply, thank you so much for your work!

**Comment by XGGNet** [comment]
> Oh, I'm sorry, this is my problem, I'll try again after revising, thank you very much for your reply, thank you so much for your work!

No worry & Thanks!

**Comment by XGGNet** [comment]
The model checkpoints is uploaded. The code's final version was refactored for readability, potentially introducing inconsistencies with the original. Plz feel free to contact if meeting further issues.

**Comment by BeiZhiRen** [comment]
> The model checkpoints is uploaded. The code's final version was refactored for readability, potentially introducing inconsistencies with the original. Plz feel free to contact if meeting further issues.

Okay, I get it, but there is a small issue. The link to your GLAS checkpoint and the link to the CVC dataset are reversed. You can make some changes when you have time. Thank you very much for your reply and for your work. In the future, I will try my best to cite and promote this work.

**Comment by XGGNet** [comment]
Oh, mine. Thank you for your credit! 😀
